google/cloud/spanner/row_test.cc

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/spanner/row.h"
+#include "google/cloud/spanner/version.h"
```

Comment:
Hmmm, why do we need to include this?

Reply:
For the SPANNER_CLIENT_NS macro?

Reply:
Feel free to ignore me, but we already include `spanner/row.h`, the macro *must* be defined by that header or the types there are going to be all wrong.  I guess I do not have IWYU religion...